code/modules/hydroponics/grown/citrus.dm

```diff
@@ -77,47 +77,3 @@
 	desc = "When life gives you lemons, be grateful they aren't limes."
 	icon_state = "lemon"
 	filling_color = "#FFD700"
-
-// Money Lemon
-/obj/item/seeds/cash
-	name = "pack of money seeds"
-	desc = "When life gives you lemons, mutate them into cash."
-	icon_state = "seed-cash"
-	species = "cashtree"
-	plantname = "Money Tree"
-	product = /obj/item/weapon/reagent_containers/food/snacks/grown/shell/moneyfruit
-	growing_icon = 'icons/obj/hydroponics/growing_fruits.dmi'
-	icon_grow = "lime-grow"
-	icon_dead = "lime-dead"
-	lifespan = 55
-	endurance = 45
-	yield = 4
-	reagents_add = list("nutriment" = 0.05)
-	rarity = 50  // Nanotrasen approves...
-
-/obj/item/weapon/reagent_containers/food/snacks/grown/shell/moneyfruit
-	seed = /obj/item/seeds/cash
-	name = "Money Fruit"
-	desc = "Looks like a lemon with something bulging from the inside."
-	icon_state = "moneyfruit"
-	bitesize_mod = 2
-
-/obj/item/weapon/reagent_containers/food/snacks/grown/shell/moneyfruit/add_juice()
-	..()
-	switch(seed.potency)
```

Comment:
This is the entire issue here why not overhaul it so up to 50 is just a larger stack of spacecash. E.g. 2 for each potency. And then past 50 instead it becomes stacks of c10s that become larger stacks at each point? E.g. Starts at 10 credit then goes up with a extra 10 c in each potency level.